docs/user/whats-new.asciidoc

```diff
@@ -1,26 +1,190 @@
+includes Logs, Metrics, APM, and Uptime. The sidebar also includes
+*Recently viewed* to easily
+access previous projects.
+
+Hidden by default, the sidebar is accessible
+with a single click on the hamburger menu.  Use the *Dock Navigation* link
+at the bottom of the sidebar to keep the navigation visible.
+
+[role="screenshot"]
+image:images/whats-new/7-8-navigation.png[Kibana side navigation organized by category]
+
+[float]
+[[dashboard-first-7-8]]
+=== Dashboard first experience
+
+In 7.7, {kibana-ref}/dashboard-create-new-dashboard.html[dashboards] became a starting point for
+creating visualizations.
+Building on that experience, this release introduces a simplified workflow
+for editing a dashboard.
+You can now open the visualization editor directly from the dashboard, and then
+automatically return to the dashboard when you save the changes.
+This workflow keeps you focused on analyzing the data.
+
+[role="screenshot"]
+image:images/whats-new/7-8-dashboard-first.png[Edit visualization from dashboard]
+
+[float]
+[[dashboard-to-dashboard-7-8]]
+=== Dashboard-to-dashboard drilldowns
+
+With 7.8, {kib} delivers {kibana-ref}/drilldowns.html[drilldowns], which enable you to create
+custom navigation paths between dashboards. Drilldowns
+work well when you have multiple dashboards that support different audiences
+or provide different levels of analysis. For example, you might
+have a dashboard that shows the overall status of multiple data centers,
+and then create a drilldown that navigates to a dashboard for a single data center.
+Drilldown paths are available to all users who view the dashboard.
+
+[role="screenshot"]
+image::drilldowns/images/drilldown_on_piechart.gif[Drilldown on pie chart that navigates to another dashboard]
+
+[float]
+[[dashboard-clone-7-8]]
+=== Cloning of dashboard elements
+{kib} now lets you {kibana-ref}/dashboard-create-new-dashboard.html#cloning-a-panel[create an identical copy] of
+any panel on the dashboard.
+You can modify the visualization in the new panel as needed, and then save your changes.
+Cloning enables you to quickly view your data from a
+new perspective.
+
+[role="screenshot"]
+image:images/clone_panel.gif[clone panel]
+
+
+[float]
+[[lens-7-8]]
+=== New visualizations for Lens
+
+{kibana-ref}/lens.html[Lens] introduces three new visualization types: pie chart, donut chart, and treemap.  New
+to {kib}, a treemap provides a hierarchical view of your data.
+Treemaps are good for visualizing part-to-whole relationships.
+
+[role="screenshot"]
+image:images/whats-new/7-8-treemap.png[Treemap visualization]
+
+
+[float]
+[[canvas-7-8]]
+=== Menu updates for Canvas
+
+{kibana-ref}/lens.html[Canvas] has a new top-level menu that makes it easier for you to find the actions you need.
+This menu provides access to charts and maps,
+so you can quickly embed them in your workload.
+The menu also includes actions for viewing, editing, and sharing your data.
```

Comment:
```suggestion
```

Reply:
These options have always been there, they are now just easier to find with the new menu.